go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/osmosis-labs/osmosis/v24
 
-go 1.21
+go 1.21.4
+
+toolchain go1.22.0
```

Comment:
Could we just bump to 1.22?

Reply:
I think we should do this bump in it's own PR if we decide to do it, and add a CHANGELOG specifically for this